rules/check-domains-against-connection-aliases.md

```diff
@@ -22,7 +22,7 @@ function (user, context, callback) {
   
   // Access allowed if domain is found
   const userEmailDomain = user.email.split('@')[1].toLowerCase();
-  const domainFound = connectionOptions.domain_aliases.some(function (domain) {
+  const domainFound = user.email_verified && connectionOptions.domain_aliases.some(function (domain) {
```

Comment:
This rule is being re-written so I'll include this change but semantically it doesn't make sense to assign `domainFound` to `false` because `user.email_verified` could be false